src/imp/imp_board.cpp

```diff
@@ -988,6 +992,43 @@ std::string ImpBoard::get_hud_text(std::set<SelectableRef> &sel)
         }
     }
     trim(s);
+
+    // Display the delta if two items of these types are selected
+    for (const ObjectType type : {ObjectType::BOARD_PACKAGE, ObjectType::BOARD_HOLE, ObjectType::VIA}) {
+        if (sel_count_type(sel, type) == 2) {
+            // Already added for packages
+            if (type != ObjectType::BOARD_PACKAGE) {
+                s += "\n\n<b> 2 " + object_descriptions.at(type).name_pl + "</b>";
```

Comment:
```suggestion
                s += "\n\n<b>2 " + object_descriptions.at(type).name_pl + "</b>";
```

No space needed here